jmclient/jmclient/maker.py

```diff
@@ -254,6 +255,23 @@ def modify_orders(self, to_cancel, to_announce):
                     self.offerlist.remove(oldorder_s[0])
             self.offerlist += to_announce
 
+    def freeze_timelocked_utxos(self):
+        """
+        Freeze all wallet's timelocked UTXOs. These cannot be spent in a
+        coinjoin because of protocol limitations.
+        """
+        if not hasattr(self.wallet_service.wallet, 'FIDELITY_BOND_MIXDEPTH'):
+            return
+
+        md_utxos = self.wallet_service.get_utxos_by_mixdepth()
+        for tx, details in md_utxos[self.wallet_service.FIDELITY_BOND_MIXDEPTH]:
+            if self.wallet_service.is_timelocked_path(details['path'])\
+                    and not self.wallet_service.is_disabled(details['path']):
```

Comment:
You can fix this with `self.wallet_service._utxos.is_disabled(*tx)`, or something similar.